www/docs/customization/docker.md

```diff
@@ -279,3 +279,52 @@ dockers:
     build_flag_templates:
     - "--builder=heroku/buildpacks:20"
 ```
+
+## How to override hardcoded registry and image name ?
```

Comment:
not so sure if this should be here or in the cookbooks section... thoughts?

Reply:
yeah, if you think that cookbooks page is the right place to put this, then we can move it over there.